backend/file_management/views.py

```diff
@@ -113,7 +114,7 @@ def delete(self, request: HttpRequest) -> Response:
         file_name: str = document.document_name
         tool_id: str = serializer.validated_data.get("tool_id")
         file_path = FileManagerHelper.handle_sub_directory_for_tenants(
-            request.org_id,
+            UserSessionUtils.get_organization_id(request),
```

Comment:
@muhammad-ali-e what's the difference between `request.org_id` and `UserSessionUtils.get_organization_id(request)`?
Can we set `request.org_id` in the middleware itself and continue to use the same?

Reply:
We've transitioned to utilizing Django session instead of storing data in the request model. Session data persists across requests, ensuring consistency. This eliminates the need to repeatedly store data in the request model with each call.

Reply:
There we are not using request.org.  they are public urls .